### ISSUE TITLE
Looks like lib/ dir isn't update(Cloud module did not built).

### ISSUE THREAD

**Issue by satoshicano** [uetchy/nature-remo#132]
**Describe the bug**
In v1.2.1, it seems to be Cloud module was not built(running tsc command)
So this patch #130 did not apply yet.

**To Reproduce**

1. (in nodejs project) add package

```sh
yarn add nature-remo@1.2.1
```

2. Look build module file

```sh
cat ./node_modules/nature-remo/lib/Cloud.js | grep -A 4 sendSignal
``` 

3. The fix has not been incorporated.

```
❯ cat ./node_modules/nature-remo/lib/Cloud.js | grep -A 4 sendSignal

    sendSignal(signalId) {
        return __awaiter(this, void 0, void 0, function* () {
            const response = yield this._post(`/1/signals/${signalId}`);
            return response;
        });
```

**Expected behavior**

Expected output is here.

```
❯ cat ./node_modules/nature-remo/lib/Cloud.js | grep -A 4 sendSignal

    sendSignal(signalId) {
        return __awaiter(this, void 0, void 0, function* () {
            const response = yield this._post(`/1/signals/${signalId}/sends`);
            return response;
        });
```

Can you check if you are building?

**Additional context**

**Comment by uetchy** [comment]
Thank you for the report.
This should have been included in 1.2.1 but hadn't.
Published new version [v1.2.2](https://github.com/uetchy/nature-remo/releases/tag/v1.2.2).

**Comment by satoshicano** [comment]
Thank you for publishing new version.
I confirmed that it works with the new version.
